js/mail.js

```diff
@@ -1196,6 +1187,35 @@ var Mail = {
 $(document).ready(function() {
 	Mail.UI.initializeInterface();
 
+	// Enables infinite scroll ability.
+	setInterval(function() {
+
+		// Element identifies the element we want to trigger our loading on.
+		//var element = $('.mail_message_summary:nth-last-child(1)');
+		var element = $('#load-more-mail-messages');
+		var outerHeight = element.outerHeight();
+		var offset = element.offset();
+		var scrollLength = $('#mail_messages').scrollTop();
+		var screenHeight = $(document).height();
+		var messageListHeight = scrollLength + offset.top + outerHeight;
+
+		if ((scrollLength + screenHeight) >= messageListHeight) {
+			Mail.UI.messageView.loadMessages('addMore');
+		}
+
+	}, 1000);
```

Comment:
> @DeepDiver1975 You would be willing to implement hundreds of lines of code from a new library over several lines of code. Then you call this elegant solution kung-fu in comparison?

yes - because the timers have the bad habit to consume cpu even if the browser/tab is not active. We shall avoid timers/loops (yes I'm aware that I'm using a timer myself - I want to remove that as well).